docs/source/examples/formulas.rst

```diff
@@ -0,0 +1,13 @@
+Formulas
+=================================
+
+.. manim:: Formula1
+    :quality: medium
+    :save_last_frame:
+
+    class Formula1(Scene):
+        def construct(self):
+            t = TexMobject(r"\int_a^b f'(x) dx = f(b)- f(a)")
```

Comment:
Similar comment as before: I'd rather expect this to be in the reference manual, in the docstring of `MathTex`.

```suggestion
            t = MathTex(r"\int_a^b f'(x) dx = f(b)- f(a)")
```